core/src/test/scala/org/apache/spark/deploy/StandaloneDynamicAllocationSuite.scala

```diff
@@ -395,8 +395,8 @@ class StandaloneDynamicAllocationSuite
     }
 
     var apps = getApplications()
-    // kill executor 1
-    assert(sc.killExecutor(executors.head))
+    // kill executor 1, and actually nothing to kill
+    assert(sc.killExecutor(executors.head) === false)
```

Comment:
because this one is killed in replacement part. 
```java
assert(executors.size === 2)
    // kill executor 1, and replace it
    assert(sc.killAndReplaceExecutor(executors.head)) //executors.head is killed here with replace = true.